### ISSUE TITLE
Unable to manually add items to shopping list

### ISSUE THREAD

**Issue by mikemathewson** [phantomate/Untare#74]
### Bug Summary

I am unable to add items directly to the shopping list (i.e. not through a recipe)

### Reproduction Steps

Go to shopping list
Click the plus icon
Search for a food to add
Select from the suggested results

### Expected Results

It should add the selected item to the shopping list

### Actual Results

Clicking on the first item does nothing, and clicking on any other item closes out of the add screen and takes you back to the shopping list.

### Checklist

- [X] I have searched the open issues for duplicates.

**Comment by phantomate** [comment]
Hi, do you still have this issue? I can't reproduce this. What Tandoor version are you using?

**Comment by mikemathewson** [comment]
I just updated Tandoor to the latest version (1.5.17) and I still experience this bug.

After more testing I have figured out exactly what is happening. When I click on one of the suggested options, that dropdown list disappears and a touch event is registered underneath the dropdown list where I touched.

So for example, if I'm in advanced mode, clicking on the options will usually end up focusing one of the fields which were being covered up by the list (i.e. Quantity, Unit, and Category).

Clicking a lower item in the list triggers a click outside of the dialog and closes the whole thing.
